packages/flipper_login/lib/text_view.dart

```diff
@@ -0,0 +1,19 @@
+library flipper_text;
+
+import 'package:flutter/material.dart';
+
+class TextView extends StatelessWidget {
```

Comment:
I like the re-usability here

Reply:
For country picker, this library was used in the old app and can be used [Country Picker](https://pub.dev/packages/country_picker)